packages/upgrade/transforms/imports-to-unified-package.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Rewrites imports from 'carbon-components-react' to '@carbon/react'
+ *
+ * Transforms:
+ *
+ * import { Button } from 'carbon-components-react';
+ * import { Tile, Tooltip } from 'carbon-components-react';
+ * import CodeSnippet from 'carbon-components-react';
+ *
+ * Into:
+ *
+ * import { Button } from "@carbon/react";
+ * import { Tile, Tooltip } from "@carbon/react";
+ * import CodeSnippet from "@carbon/react";
```

Comment:
Is an important part of this codemod that it will transform multiple imports, or are we assuming they will only have one?

Reply:
@joshblack My assumption is that they'll have only one, but the codemod will transform multiples if they have them. 

These comments were more to show that different types of imports are supported. Importing default, importing a single named, and importing multiple named are all handled appropriately. I can cut this comment down to be more clear.

Reply:
Updated